bridge-impl/src/main/java/com/liferay/faces/bridge/application/internal/ViewHandlerImpl.java

```diff
@@ -125,6 +126,17 @@ public String getActionURL(FacesContext facesContext, String viewId) {
 	}
 
 	@Override
+	public String getResourceURL(FacesContext fc, String string) {
+
+		if (PortletResourceUtil.isPortletResourceURL(string)) {
```

Comment:
@ngriffin7a, I believe this change is valid because it is essentially the same change that was made inside Mojarra's `RenderKitUtils` from [JSF 2.1](https://github.com/javaserverfaces/mojarra/blob/2.1.29-05/jsf-ri/src/main/java/com/sun/faces/renderkit/RenderKitUtils.java#L1356-L1357) to [2.2](https://github.com/javaserverfaces/mojarra/blob/2.2.13/jsf-ri/src/main/java/com/sun/faces/renderkit/RenderKitUtils.java#L1403-L1404):

JSF 2.1:

```
// public static final String RESOURCE_IDENTIFIER = "/javax.faces.resource";
if (value.contains(ResourceHandler.RESOURCE_IDENTIFIER)) {
    return value;
```

JSF 2.2

```
if (handler.isResourceURL(value)) {
    return value;
```

The change in Mojarra made this code work correctly in JSF 2.2 since we override `ResourceHandler.isResourceURL()`:

```
@Override
public boolean isResourceURL(String url) {
	return (url != null) && url.contains("javax.faces.resource=");
}
```